services/src/main/java/org/jboss/windup/web/services/model/ApplicationGroup.java

```diff
@@ -56,9 +58,11 @@
     private AnalysisContext analysisContext;
 
     @OneToMany(fetch = FetchType.EAGER, mappedBy = "applicationGroup")
+    @Fetch(FetchMode.SELECT)
     private Set<RegisteredApplication> applications;
 
     @OneToMany(fetch = FetchType.EAGER, mappedBy = "group")
+    @Fetch(FetchMode.SELECT)
```

Comment:
I assume that it is a hibernate bug? Loading it as a join seemed to cause duplicated rows and I doubt the spec allows that. I think this forces it to use a separate query.